doc/multiple_languages.md

```diff
@@ -108,7 +108,7 @@ creates PDFs for every problem language statement `problem.xy.tex`.
 With arguments,
 
 ```sh
-bt pdf --language en --language fr
+bt pdf --languages en fr
```

Comment:
not sure changing `--language` to `--languages` makes that much sense when probably the usual case is to pass 1 lang only, but OK.
Could also rename to `--lang` maybe?

Reply:
That comment actually belongs to #441 :see_no_evil: Looks like the `absolute2` branch was based onto the `legacy-export` branch. But yeah, I'm fine with `--lang` (I'll change it there) :slightly_smiling_face: